---a/net/minecraft/world/EnumSkyBlock.java
+++b/net/minecraft/world/EnumSkyBlock.java
@@ -1,16 +1,13 @@
 package net.minecraft.world;
 
-public enum EnumSkyBlock
-{
+public enum EnumSkyBlock {
     SKY(15),
     BLOCK(0);
     public final int defaultLightValue;
 
     private static final String __OBFID = "CL_00000151";
 
-    private EnumSkyBlock(int p_i1961_3_)
-    {
+    private EnumSkyBlock(int p_i1961_3_) {
         this.defaultLightValue = p_i1961_3_;
     }
-}
+}
